outputs.tf

```diff
@@ -1,20 +1,15 @@
 locals {
   grafana_endpoint = join("", module.grafana.*.grafana_endpoint)
 }
+
 output "cluster_id" {
   description = "Cluster ID of Thanos deployment"
   value       = local.postfix_name
 }
 
 output "thanos_space_id" {
   description = "Cloud foundry space ID of Thanos"
-  value       = cloudfoundry_space.space.id
-
-   depends_on = [
```

Comment:
This depends on is still needed if we create a space. We need to make sure the permissions are allocated before the space id is returned for other usages. Otherwise we have a race condition.

Reply:
Yeah, space creation will be ripped out as dealing with the race condition on so many levels and trying to be compatible with both uses cases is not worth the complexity